src/osversion/VersionTranslator.cpp

```diff
@@ -7897,6 +7898,83 @@ namespace osversion {
       {"OS:Coil:WaterHeating:AirToWaterHeatPump:Wrapped", 13},
     }};
 
+    // Making the map case-insentive by providing a Comparator `IstringCompare`
```

Comment:
insensitive

Reply:
I'll fix it in the next PR that rename OS:DistictHeating to OS:DistrictHeating:Water and takes in your DistrictHeating:Steam PR. Trying to save CI cycles.
(this is also a copy paste, I made that typo a long time ago :) )